test/unit/transform/classes_test.py

```diff
@@ -347,19 +347,21 @@
         (5, pd.NA, pd.NA, -5.0, "barry"),
         (6, 2000, "2000-01-01", 231.1, "replace me"),
         (7, pd.NA, pd.NA, 101.10, pd.NA),
-        (8, 2012, "01-01-2020", 899.98, "another plant name"),
```

Comment:
pandas 2.0 really does not like this syntax. We can support it but would want to call out the specific row that needs to be cast in this format. Are we expected to support this for any of the input data?

Reply:
I don't know if this scenario (wildly varying date formats in the same column) comes up anywhere in our actual data -- and if it does, we'll need to do the `pd.to_datetime(format="mixed")` wherever it happens, so I suspect the test data should probably be altered to avoid this problem.

Reply:
Thanks Zane. Since I did indeed change the test data to avoid this problem, I'm interpreting your comment to mean that you approve of the way I handled this problem by changing the test data.